### PR TITLE
Add filletblade

### PR DESCRIPTION
**Pull request by tzwang1**

### CODE REVIEW

File: internal/weapons/sword/filletblade/filletblade.go [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+	icd := 0
+	cd := 960 - 60*r
+
+	c.Events.Subscribe(core.OnDamage, func(args ...interface{}) bool {
+		atk := args[1].(*core.AttackEvent)
+
+		if atk.Info.ActorIndex != char.CharIndex() {
+			return false
+		}
```
Comment (shizukayuki) [inline internal/weapons/sword/filletblade/filletblade.go:26]: missing active char check here. fillet blade does not work off-field.

File: internal/weapons/sword/filletblade/filletblade.go [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+			ActorIndex: char.CharIndex(),
+			Abil:       "Fillet Blade Proc",
+			AttackTag:  core.AttackTagWeaponSkill,
+			ICDTag:     core.ICDTagNone,
+			ICDGroup:   core.ICDGroupDefault,
+			Element:    core.Physical,
+			Durability: 100,
+			Mult:       2.0 + 0.4*float64(r),
+		}
+		c.Combat.QueueAttack(ai, core.NewDefCircHit(2, false, core.TargettableEnemy), 0, 1)
```
Comment (shizukayuki) [inline internal/weapons/sword/filletblade/filletblade.go:45]: i think the radius needs to be changed to `0.1`.

**Review by shizukayuki — CHANGES_REQUESTED** [review]
everything else looks fine to me.

File: internal/weapons/sword/filletblade/filletblade.go (hunk 1 above)
Comment (tzwang1) [reply internal/weapons/sword/filletblade/filletblade.go:26]: Added check

File: internal/weapons/sword/filletblade/filletblade.go (hunk 2 above)
Comment (tzwang1) [reply internal/weapons/sword/filletblade/filletblade.go]: Changed radius to 0.1. Btw how can I tell what the radius should be?

**Review by tzwang1 — COMMENTED** [review]
Thanks for the review!

**Review by shizukayuki — APPROVED** [review]

File: internal/weapons/sword/filletblade/filletblade.go (hunk 2 above)
Comment (shizukayuki) [reply internal/weapons/sword/filletblade/filletblade.go]: either eyeball it or check the wiki.
https://genshin-impact.fandom.com/wiki/Area_of_Effect#Comparison.5B17.5D

**Pull request merged**